react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+class StopWatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isClicked: false,
+      timer: 0,
+      interval: null
+    };
+    this.handleClick = this.handleClick.bind(this);
+    this.playWatch = this.playWatch.bind(this);
+    this.pauseWatch = this.pauseWatch.bind(this);
+    this.resetWatch = this.resetWatch.bind(this);
+  }
+
+  handleClick() {
+    if (!this.state.isClicked) {
+      this.playWatch();
+    } else {
+      this.pauseWatch();
+    }
+  }
+
+  playWatch() {
+    const intervalCount = setInterval(() => {
+      this.setState({ isClicked: this.state.isClicked, timer: this.state.timer + 1 });
+    }, 1000);
+    this.setState({ isClicked: true, timer: this.state.timer, interval: intervalCount });
+  }
+
+  pauseWatch() {
+    clearInterval(this.state.interval);
+    this.setState({ isClicked: false, timer: this.state.timer });
```

Comment:
Here you just need to set `isClicked` to false, no need to set timer to itself here.